crates/lib/src/executable.rs

```diff
@@ -236,8 +236,7 @@ impl<'executable> Executable<'executable, '_> {
     }
 }
 
-type ExecuteResultQVM = Result<execution_data::Qvm, Error>;
-type ExecuteResultQPU = Result<execution_data::Qpu, Error>;
+type ExecuteResult = Result<execution_data::ExecutionData, Error>;
```

Comment:
Could we change this to `ExecutionResult`? `ExecuteResult` seems off to me somehow, like we are executing the result instead of it being the result of an execution.

Reply:
Yeah, that is weird, changed it.